addons/sourcemod/scripting/nd_discord_playerlist.sp

```diff
@@ -0,0 +1,193 @@
+#include <sdktools>
+#include <nd_stocks>
+#include <nd_team_eng>
+
+Handle SaveTeamsTimer;
+bool SavingTeams;
+Handle DatabaseHandle;
+char PlayerNames[MAXPLAYERS + 1][MAX_NAME_LENGTH];
+
+public Plugin myinfo =
+{
+    name        = "[ND] Discord Player List",
+    author      = "zookatron",
+    description = "Provides player list information to Discord",
+    version     = "dummy",
+    url         = "https://github.com/stickz/Redstone/"
+};
+
+public void OnPluginStart()
+{
+    if (!SQL_CheckConfig("discord"))
+    {
+        SetFailState("Could not find configuration for \"discord\" database");
+        return;
```

Comment:
Anything after the SetFailState is not executed so we don't need the return here.

Reply:
done